### PR TITLE
Harden RBAC configuration for CCM

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Add rbac to CCM manifest example

### CODE REVIEW

**Comment by falzm** [comment]
@pierre-emmanuelJ you can rebase your branch on master to re-run the integration tests 👍

File: docs/examples/cloud-controller-manager.yml [hunk 1]
```diff
@@ -67,11 +67,76 @@ metadata:
...
+  verbs:
+  - create
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - '*'
```
Comment (7fELF) [inline docs/examples/cloud-controller-manager.yml:90]:
```suggestion
  - '*'
  - apiGroups:
  - ""
  resources:
  - configmaps
  verbs:
  - 'get'
```

File: docs/examples/cloud-controller-manager.yml [hunk 2]
```diff
@@ -67,11 +67,82 @@ metadata:
...
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - 'get'
```
Comment (7fELF) [inline docs/examples/cloud-controller-manager.yml:96]:
```suggestion
  - 'get'
  - 'watch'
```

File: integtest/manifests/deployment.yml [hunk 3]
```diff
@@ -65,14 +65,78 @@ metadata:
...
+  verbs:
+  - create
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - '*'
```
Comment (7fELF) [inline integtest/manifests/deployment.yml:90]:
```suggestion
  - '*'
- apiGroups:
  - ""
  resources:
  - configmaps
  verbs:
  - 'get'
  - 'watch'
```

**Comment by pierre-emmanuelJ** [comment]
We should rebase this branch on master to have the lastest integration test version

**Comment by pierre-emmanuelJ** [comment]
Note: rebased on master
@7fELF

**Review by 7fELF — APPROVED** [review]

**Review by falzm — APPROVED** [review]
LGTM 👍

**Pull request merged**